build.yaml

```
@@ -11,6 +11,15 @@ description: "Generate reports of your media library"
 category: "General"
 artifacts:
   - "Jellyfin.Plugin.Reports.dll"
+  - "ClosedXML.dll"
+  - "DocumentFormat.OpenXml.dll"
+  - "ExcelNumberFormat.dll"
+  - "Microsoft.Win32.SystemEvents.dll"
+  - "System.Drawing.Common.dll"
+  - "System.IO.Packaging.dll"
+  - "runtimes/unix/lib/netcoreapp2.0/System.Drawing.Common.dll"
+  - "runtimes/win/lib/netcoreapp2.0/Microsoft.Win32.SystemEvents.dll"
+  - "runtimes/win/lib/netcoreapp2.0/System.Drawing.Common.dll"
```

```suggestion
  - "System.IO.Packaging.dll"
```

The removed are already bundled with Jellyfin. Unsure if System.IO.Packaging is required- will hopefully have time to test later